### ISSUE TITLE
VagrantCloud W10 virtualbox missing

### ISSUE THREAD

**Issue by mkevenaar** [StefanScherer/packer-windows#308]
The v2021.10.06 release of windows_10 for VirtualBox is missing:

https://app.vagrantup.com/StefanScherer/boxes/windows_10/versions/2021.10.06/providers/virtualbox.box

This download returns in a 404

**Comment by StefanScherer** [comment]
BTW I‘ve uploaded a windows_11 box. Maybe you can give it a try, because I don‘t know how long I will upload Windows 10 boxes. 😅

**Comment by melkio** [comment]
Hi @StefanScherer  and thanks for your work.
I'm using your boxes for a long time.

I don't know why but latest releases for Windows10 and Windows11 boxes are not available for VirtualBox provider.
Looking to circleci link you shared I noted this message:

```
Create a new version 2021.10.06 for windows_10 in Vagrant Cloud
{"errors":["Version has already been taken (2021.10.06)"],"success":false}
```

Could be something related to missing boxes? I'm not a packer expert. I could be completely wrong :)

**Comment by StefanScherer** [comment]
Thank you @melkio :-)

Looks like the build scripts and CI pipeline needs some care. I see this warning/error:

```
==> virtualbox-iso: Error unregistering floppy: VBoxManage error: VBoxManage: error: Could not find a controller named 'Floppy Controller'. Not considering this a critical failure; build will continue.
```

And the latest CI builds have a timeout in CircleCI https://app.circleci.com/pipelines/github/StefanScherer/packer-builder/669/workflows/8842c015-cb49-4102-b9ef-820a03f5227f/jobs/1984
It only uploaded the VMware box.

Seems like CircleCI changed their policies for open source / free users. Although I have still free build minutes this month, there seems to be a timeout after 60 minutes and that exceeds the whole packer build and upload. That will take a bit of time to investigate this which I don't have right now 😬

**Comment by basictheprogram** [comment]
Sorry to pollute this issue but my question is is towards your [comment](https://github.com/StefanScherer/packer-windows/issues/308#issuecomment-942248929) regarding Windows10. 

Can you let us know when you no longer will build Windows10?

Thanks.

**Comment by StefanScherer** [comment]
@basictheprogram I'm trying to build Windows 10 for a while, but yes I can let you know.
I try to find some time to investigate the CircleCI problem and maybe move over to GitHub Actions.

**Comment by StefanScherer** [comment]
@melkio @mkevenaar @basictheprogram Please try again. I've moved my pipeline from CircleCI to GitHub Actions and created fresh Windows 10 and Windows 11 boxes for VMware and VirtualBox.

**Comment by basictheprogram** [comment]
Don't know if you want a new issue. I'll append to this issue.
```
$ vagrant box  update
==> default: Checking for updates to 'StefanScherer/windows_10'
    default: Latest installed version: 2021.08.17
    default: Version constraints:
    default: Provider: virtualbox
==> default: Box 'StefanScherer/windows_10' (v2021.08.17) is running the latest version.
```
```
Vagrant.configure("2") do |config|
  config.vm.box = "StefanScherer/windows_10"
end
```

**Comment by StefanScherer** [comment]
Thanks, outch I forgot to press the release button. It's live now!

**Comment by basictheprogram** [comment]
Just confirming.

```
$ vagrant box outdated
Checking if box 'StefanScherer/windows_10' version '2021.08.17' is up to date...
A newer version of the box 'StefanScherer/windows_10' for provider 'virtualbox' is
available! You currently have version '2021.08.17'. The latest is version
'2021.12.09'. Run `vagrant box update` to update.
```
